torchtune/training/_compile.py

```diff
@@ -77,6 +79,10 @@ def compile_loss(loss: nn.Module, verbose: bool = True) -> nn.Module:
         )
     elif isinstance(loss, ForwardKLWithChunkedOutputLoss):
         loss.fkl_loss = torch.compile(loss.fkl_loss, backend=backend)
+    elif isinstance(loss, ReverseKLWithChunkedOutputLoss):
+        loss.rkl_loss = torch.compile(loss.rkl_loss, backend=backend)
+    elif isinstance(loss, SymmetricKLWithChunkedOutputLoss):
+        loss.sym_kl_loss = torch.compile(loss.sym_kl_loss, backend=backend)
```

Comment:
Kinda related to my comment above, we should also consider just defining a protocol `ChunkedLoss` or something having a method e.g. `def base_loss(self, *args, **kwargs)` and having all of `CEWithChunkedOutputLoss`, `ForwardKLWithChunkedOutputLoss`, `ReverseKLWithChunkedOutputLoss`, `SymmetricKLWithChunkedOutputLoss` inherit from that. Then L76-L85 basically just become 

```
if isinstance(loss, ChunkedLoss):
	loss.base_loss = torch.compile(loss.base_loss, backend=backend)
```

Again, not a blocker for this particular PR.